Jenkinsfile

```diff
@@ -168,18 +171,55 @@ pipeline {
                     expression { return params.DEPLOY }
                 }
             }
-            steps {
-                postGitHubStatus("jenkins/functional-tests", "pending", "Started", env.RUN_DISPLAY_URL)
-
-                script {
-                    LAST_STAGE = env.STAGE_NAME
-                    timeout(time: 60, unit: 'MINUTES') {
-                        // sh "docker-compose -f docker-compose.e2e.yml run e2e -e CYPRESS_baseUrl=https://${CFGOV_HOSTNAME}"
-                        sh "docker run -v ${WORKSPACE}/test/cypress:/app/test/cypress -v ${WORKSPACE}/cypress.json:/app/cypress.json -w /app -e CYPRESS_baseUrl=https://${CFGOV_HOSTNAME} -e CI=1 cypress/included:6.8.0 npx cypress run -b chrome --headless"
+            parallel {
+                stage('Cypress admin tests') {
+                    options {
+                        timeout(time: 10, unit: 'MINUTES')
+                    }
+                    steps {
+                        sh "./docker-compose -f docker-compose.e2e.yml run admin-tests"
+                    }
+                }
+                stage('Cypress component tests') {
+                    options {
+                        timeout(time: 10, unit: 'MINUTES')
+                    }
+                    steps {
+                        sh "./docker-compose -f docker-compose.e2e.yml run component-tests"
+                    }
+                }
+                stage('Cypress consumer tools tests') {
+                    options {
+                        timeout(time: 10, unit: 'MINUTES')
+                    }
+                    steps {
+                        sh "./docker-compose -f docker-compose.e2e.yml run consumer-tools-tests"
+                    }
+                }
+                stage('Cypress data research tests') {
+                    options {
+                        timeout(time: 10, unit: 'MINUTES')
+                    }
+                    steps {
+                        sh "./docker-compose -f docker-compose.e2e.yml run data-research-tests"
+                    }
+                }
+                stage('Cypress paying for college tests') {
+                    options {
+                        timeout(time: 10, unit: 'MINUTES')
+                    }
+                    steps {
+                        sh "./docker-compose -f docker-compose.e2e.yml run paying-for-college-tests"
+                    }
+                }
+                stage('Cypress rules-policy tests') {
+                    options {
+                        timeout(time: 10, unit: 'MINUTES')
+                    }
+                    steps {
+                        sh "./docker-compose -f docker-compose.e2e.yml run rules-policy-tests"
                     }
                 }
-
-                postGitHubStatus("jenkins/functional-tests", "success", "Passed", env.RUN_DISPLAY_URL)
```

Comment:
Is there a reason we want to remove the posting of status back to the PR?

Reply:
Good catch. I added status posting back to Jenkinsfile.